client/assets/style.css

```diff
@@ -0,0 +1,138 @@
+/* HEADER START */
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    float: left;
+}
+  
+li {
+    float: left;
+    margin-left: 5px;
+}
+  
+nav a {
+    display: block;
+    color: #262626;
+    padding: 5px 5px;
+    text-decoration: none;
+    font-size: 14px;
+    font-family: arial,sans-serif;
+    padding: 5px;
+    margin: 5px;
+    margin-top: 10px;
+}
+
+.rightnavbar {
```

Comment:
consider using kebab casing for reading purposes. Its common use for CSS. 
e.g
right-nav-bar